### PR TITLE
added team positions

### PR DESCRIPTION
**Pull request by ghost**
added team positions, tried to get everyone in there equal amount of times, but i think one person is only once (fiona), if you have objections, comment or commit changes

### CODE REVIEW

**Review by cjfarrelly — APPROVED** [review]
lgtm!

Changing response after seeing slack!

File: content/process.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ This page will outline the various processes that we have refined as the project
...
 Being a forward thinking company, we like to keep our people both happy and challenged, this means we change roles **A LOT!**
 
-| Role | Week 1 | Week 2 | Week 3 | Week 4 | Week 5 |
-|---|---|---|---|---|---|
-| Scrum Master ||||||
-| Proxy PO ||||||
-| Tech Lead ||||||
+| Role | Week 0 1/3 | Week 1 8/3 | Week 2 15/3 | Week 3 22/3 | Week 4 29/3 | Week 5 5/4 | Week 6 12/4 | Week 7 19/4 |
+|---|---|---|---|---|---|---|---|---|
+| Scrum Master | Francis | Colin | Conor  | Emmet   | Fiona | Lee | Peter   | Wes |
```
Comment (cjfarrelly) [inline content/process.md:17]: This should probably reflect what Wes has on slack

**Review by cjfarrelly — CHANGES_REQUESTED** [review]

**Review by l00073976 — COMMENTED** [review]
lgtm

**Comment by ghost** [comment]
@cjfarrelly sorry missed your change request there. do you still want it?

**Pull request merged**